base/loading.jl

```diff
@@ -591,9 +591,15 @@ function explicit_manifest_uuid_path(project_file::String, pkg::PkgId)::Union{No
             return path
         end
         hash === nothing && return nothing
+        # Look for bundled packages in `$project/packages`
+        depot_path = DEPOT_PATH
+        project = active_project()
+        if project !== nothing
```

Comment:
```suggestion
        if project !== nothing && project in load_path()
```

Reply:
Could you explain this change?

Reply:
Seems weird to look for stuff in a path that is not on the LOAD_PATH.

Reply:
Sure. I think this will be true in 99% of cases anyway. The only way that the active project wouldn’t be the first entry in the load path would be if someone manually popped it off.

Reply:
Yes, so my suggestion fixes that.

Reply:
The situation where this can happen is if `@` is removed from the `LOAD_PATH`. I have to think a little about whether it makes sense for that to interact with this or not...

Then this can be simplified to this:
```suggestion
        if project in load_path()
```
That's because `load_path` already filters out `nothing` values and only returns strings.